hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -121,18 +122,44 @@ public synchronized void open() throws IOException {
         String hosts = config.get(HbaseOptions.HBASE_HOSTS);
         int port = config.get(HbaseOptions.HBASE_PORT);
         String znodeParent = config.get(HbaseOptions.HBASE_ZNODE_PARENT);
-
-        Configuration hConfig = HBaseConfiguration.create();
-        hConfig.set(HConstants.ZOOKEEPER_QUORUM, hosts);
-        hConfig.set(HConstants.ZOOKEEPER_CLIENT_PORT, String.valueOf(port));
-        hConfig.set(HConstants.ZOOKEEPER_ZNODE_PARENT, znodeParent);
-
-        hConfig.setInt("zookeeper.recovery.retry",
-                       config.get(HbaseOptions.HBASE_ZK_RETRY));
-
-        // Set hbase.hconnection.threads.max 64 to avoid OOM(default value: 256)
-        hConfig.setInt("hbase.hconnection.threads.max",
-                       config.get(HbaseOptions.HBASE_THREADS_MAX));
+        boolean isEnableKerberos = config.get(HbaseOptions.HBASE_KERBEROS_ENABLE);
+        Configuration hConfig = null;
+        if(isEnableKerberos) {
+            String krb5Conf = config.get(HbaseOptions.HBASE_KRB5_CONF);
+            System.setProperty("java.security.krb5.conf", krb5Conf);
```

Comment:
remove this line if possible.

Reply:
which line need to remobe?pls tell me line num.

Reply:
@houzhizhen It seems that setting this environment `java.security.krb5.conf ` is the best way, see [Locating the krb5.conf Configuration File](https://docs.oracle.com/javase/8/docs/technotes/guides/security/jgss/tutorials/KerberosReq.html)